ApplicationService/externalrepository.py

```diff
@@ -0,0 +1,24 @@
+from ApplicationService.externalAccount import externalAccount
+import maybe
+from ApplicationService.connection_pool import connection_pool
+from ApplicationService.threadIdentity import identity
+from ApplicationService.repositories.externalaccountsrepository import externalAccountsRepository
+
+class externalRepository(externalAccountsRepository):
+    def __init__(self, cpool: connection_pool, identifier: identity):
+        self.connection_pool = cpool
+        self.identifier = identifier
+
+    def get_by_login(self, login: str) -> maybe.maybe:
```

Comment:
https://docs.python.org/3/library/typing.html#typing.Generic

Can we use generics to refine the return of get_by_login?

Reply:
The Generic inheritance would be in maybe class or should be used in get_by_login only?

Reply:
maybe